packages/manager/src/queries/vlans.ts

```diff
@@ -2,16 +2,15 @@ import { getVlans, VLAN } from '@linode/api-v4/lib/vlans';
 import { APIError } from '@linode/api-v4/lib/types';
 import { useQuery } from 'react-query';
 import { queryPresets } from './base';
+import { getAll } from 'src/utilities/getAll';
 
 export const queryKey = 'vlans';
 
-export const _getVlans = (): Promise<VLAN[]> =>
-  getVlans().then(({ data }) => data);
+const _getVlans = (): Promise<VLAN[]> =>
+  getAll<VLAN>(() => getVlans())().then(({ data }) => data);
```

Comment:
It is crucial that the params value from the `getAll` utility gets passed to the `getVlans` function. We need to make either of the following changes for this to work

```suggestion
  getAll<VLAN>((params) => getVlans(params))().then(({ data }) => data);
```

Or 

```suggestion
  getAll<VLAN>(getVlans)().then(({ data }) => data);
```

Reply:
@bnussman-akamai I wasn't noticing the difference in the response at first, but adding the `params` is what sets the page size to 200.